### PR TITLE
adding safari_cookie definitions

### PR DESCRIPTION
**Pull request by chb2mn**

### CODE REVIEW

File: data/webbrowser.yaml [hunk 1]
```diff
@@ -1282,6 +1282,17 @@ sources:
...
 urls: ['https://forensics.wiki/apple_safari']
 ---
+name: SafariCookies
+doc: Safari Cookies database.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/Library/Cookies/*'
+    - '%%users.homedir%%/Library/Containers/com.apple.Safari/Data/Library/Cookies/*'
```
Comment (joachimmetz) [inline data/webbrowser.yaml:1292]: maybe add Cookies.binarycookies as file name?

File: data/webbrowser.yaml [hunk 2]
```diff
@@ -1282,6 +1282,17 @@ sources:
 supported_os: [Darwin, Windows]
 urls: ['https://forensics.wiki/apple_safari']
 ---
+name: SafariCookies
+doc: Safari Cookies database.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/Library/Cookies/*'
```
Comment (joachimmetz) [inline data/webbrowser.yaml:1291]: any way this glob could be made more specific?

Comment (chb2mn) [reply data/webbrowser.yaml:1291]: sure, I was trying to make it resilient to any OSX changes, but I can't find any examples that aren't `Cookies.binarycookies`

Comment (joachimmetz) [reply data/webbrowser.yaml]: thanks, that's fine we can add additional paths in the future if needed

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**